ui/templates/search/indexes/learningresources/learningresource_text.txt

```diff
@@ -0,0 +1,3 @@
+{{ object.title }}
```

Comment:
Yeah, it's non-obvious, but it's required by the [Haystack config](https://github.com/mitodl/lore/pull/141/files#diff-1b36f1f0c5b2773e5fe7a4cfdb525190R20).